src/state/accountSelectors.ts

```diff
@@ -134,6 +158,38 @@ export const getSubaccountOpenOrders = createSelector([getSubaccountOrders], (or
   )
 );
 
+/**
+ * @param state
+ * @returns list of conditional orders that have not been filled or cancelled for all subaccount positions
+ */
+export const getSubaccountConditionalOrders = createSelector(
+  [getSubaccountOpenOrders, getOpenPositions],
+  (allOpenOrders, positions) => {
+    const stopLossOrders: SubaccountOrder[] = [];
+    const takeProfitOrders: SubaccountOrder[] = [];
+
+    positions?.forEach((position) => {
+      const orderSideForConditionalOrder =
+        position?.side?.current === AbacusPositionSide.LONG
+          ? AbacusOrderSide.sell
+          : AbacusOrderSide.buy;
+      const conditionalOrders = allOpenOrders?.filter(
+        (order) => order.marketId === position.id && order.side === orderSideForConditionalOrder
+      );
```

Comment:
can we actually do something similar to `getMarketOrders` for openOrders indexed by marketId? so we can just do something like following, and not have to filter the orders every time

```
const conditionalOrders = openOrdersByMarketId[order.marketId]
...
const isCurrentOrderSide = order.side === orderSideForConditionalOrder;
if (isStopLossOrder(order) && isCurrentOrderSide)...
...
```